src/components/Api.js

```diff
@@ -0,0 +1,84 @@
+import Card from "./Card";
+
+export default class Api {
+  constructor({ baseUrl, headers }) {
+    (this._baseUrl = baseUrl), (this._headers = headers);
```

Comment:
`    (this._baseUrl = baseUrl), (this._headers = headers);
`

Please, delete `,` and `()`


It should be like that

```js
  constructor({ baseUrl, headers }) {
    this._baseUrl = baseUrl;
    this._headers = headers;
  }

```